Android/app/src/main/java/com/example/kahye/common/PostTest.java

```diff
@@ -0,0 +1,68 @@
+package com.example.kahye.common;
```

Comment:
Do we really need this PostTest.java file in our project?
I think it is not necessary.
Remove this java file.